src/scaffoldmaker/annotation/bladder_terms.py

```diff
@@ -4,9 +4,13 @@
 
 # convention: preferred name, preferred id, followed by any other ids and alternative names
 bladder_terms = [
-    ( "urinary bladder", "FMA:15900", "UBERON:0001255" ),
-    ( "neck of urinary bladder", "FMA:15912", "UBERON:0001258"),
-    ( "body of urinary bladder", None)  # needs to be replaced with an actual term e.g. urinary bladder (which includes neck)
+    ("urinary bladder", "FMA:15900", "UBERON:0001255"),
+    ("neck of urinary bladder", "FMA:15912", "UBERON:0001258"),
+    ("Dome of the Bladder", None),
+    ("Serosa of body of urinary bladder", None),
+    ("Lumen of body of urinary bladder", None),
+    ("Serosa of neck of urinary bladder", None),
+    ("Lumen of neck of urinary bladder", None)
```

Comment:
Need to put in the string with the term identifier, in this case "ILX:XXXXXXXX" since these are SPARC terms.
https://scicrunch.org/scicrunch/interlex/view/ilx_0738433
Can search for these.
https://scicrunch.org/scicrunch/interlex/search?q=serosa+bladder
Also, please use exact capitalisation of the text from interlex, e.g. Serosa and Lumen are not capitalised I think.
If you change them here, ensure they are changed everywhere.
"Dome of the Bladder" is an unusual choice, but it's in there; I would have expected "dome of urinary bladder".